### PR TITLE
ssh/tailssh: add envknob for default PATH

### PR DESCRIPTION
**Pull request by bradfitz**
As backup plan, just in case the earlier fix's logic wasn't correct and we want to experiment in the field or have users have a quicker fix.

Updates #5285

### CODE REVIEW

File: ssh/tailssh/incubator.go [hunk 1]
```diff
@@ -626,19 +641,24 @@ func pathFromPAMEnvLine(line []byte, u *user.User) (path string) {
...
 }
 
+func expandDefaultPathTmpl(t string, u *user.User) string {
+	p := strings.NewReplacer(
+		"@{HOME}", u.HomeDir,
+		"@{PAM_USER}", u.Username,
+	).Replace(t)
+	if strings.Contains(p, "@{") {
+		// If there are unknown expansions, conservatively fail closed.
+		return ""
```
Comment (maisem) [inline ssh/tailssh/incubator.go:657]: Return t?

**Review by maisem — APPROVED** [review]

File: ssh/tailssh/incubator.go (hunk 1 above)
Comment (bradfitz) [reply ssh/tailssh/incubator.go:657]: If p contains it, so will t. So they screwed up the template so we ignore it.

Comment (maisem) [reply ssh/tailssh/incubator.go:657]: I was thinking that having it show up in the PATH unexpanded would be better.
However corrupting the PATH sounds worse, so this lgtm

**Comment by ShadowJonathan** [comment]
Thank you, I think this'll suffice in case there's a problem

**Pull request merged**